Automattic/wp-calypso
# Stats Date Picker: Design Tweaks

Related to #83269

## Proposed Changes

* updating cancel and apply button spacing
* adjusting shortcut tick position by extending the width and adding more white space
* adjusting border-radius and letter spacing to match design requirements
* updating HTML order or fix tab order
* adding aria labels
* adding 2px between shortcuts to avoid clipping of an outline when hovering the next item

## Testing Instructions



* open the traffic page on the live branch
* make sure that date picker works and looks the same as on the screenshot

<img width="562" alt="SCR-20231024-mnsj" src="https://github.com/Automattic/wp-calypso/assets/112354940/44b74eee-79a6-4a73-a2d9-170ee1679e1d">

## Pre-merge Checklist
